config/config.php

```diff
@@ -384,8 +384,24 @@
 N4G2ppYlgquUXqcb6PcLPA==',
 			],
 		],
+		'17.0.2' => [
```

Comment:
Shouldnt this be something like in the 15 block instead?

Reply:
No. This is to make sure that stable users first get a bump to their latest stable version.